### DISCUSSION TITLE
Windowed vs tiled in to_raster

### DISCUSSION THREAD

**Discussion by andhuang-CLGX** [corteva/rioxarray#406] [General]
In https://corteva.github.io/rioxarray/stable/examples/dask_read_write.html: `tiled` is used, but not documented. What's the difference between tiled vs windowed?

`ds.rio.to_raster("test.tif", driver="COG", tiled=True, windowed=True)`

**Comment by snowman2** [comment]
https://corteva.github.io/rioxarray/stable/rioxarray.html#rioxarray.raster_dataset.RasterDataset.to_raster
```
windowed (bool, optional) – If True, it will write using the windows of the output raster. This is useful for loading data in chunks when writing. Does not do anything when writing with dask. Default is False.
```

`tiled` - see: https://gdal.org/drivers/raster/gtiff.html#creation-options

**Comment by andhuang-CLGX** [comment]
From that, do I need to add `NUM_THREADS` or `num_threads=8` too if I am using multiple threads?

**Reply by andhuang-CLGX** [reply]
Nevermind, that seems to be only for compression related stuff.

**Reply by andhuang-CLGX** [reply]
Nevermind on the nevermind! By specifying num_threads=4, it cut the processing time by half!!
